iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/string_internal.hpp

```diff
@@ -142,6 +151,9 @@ struct GetData<std::string>
     }
 };
 
+// AXIVION Next Construct AutosarC++19_03-A0.1.6
+// AXIVION Next Construct AutosarC++19_03-A7.5.1 : It is one of the specialization for the
```

Comment:
Can you move this down right before `return &data` and change the justification to `Used for template meta-programming and safe in this context`

Reply:
@elBoberido Sure. take care